### PR TITLE
Adding tests for gametime module

### PR DESCRIPTION
**Pull request by chason**
#### Brief overview of PR changes/additions

Adds basic tests for gametime module

#### Motivation for adding to Evennia

I want to confidently make changes to gametime and not worry about breaking anything

### CODE REVIEW

**Comment by chason** [comment]
ok that failed very strangely. These all pass locally.

**Comment by chason** [comment]
ah, it looks like something else in the test suite is affecting the gametime module. I was only testing evennia.utils locally before. I will look into this tomorrow and fix it up.

**Comment by Griatch** [comment]
It's a twisted async situation, it looks like.

**Comment by chason** [comment]
some of the tests themselves are failing too. My guess is that another test suite is changing some of the configuration/constants or monkey patching something that is causing different results. the `gametime.gametime()` function in particular is returning an integer which shouldn't be possible if it is running "naturally"

**Comment by chason** [comment]
Yay, finally all the tests pass

File: evennia/contrib/tests.py [hunk 1]
```diff
@@ -222,6 +222,8 @@ def fromtimestamp(cls, timestamp):
 
 
 @patch('evennia.contrib.extended_room.datetime.datetime', ForceUTCDatetime)
+# mock gametime to return April 9, 2064, at 21:06 (spring evening)
+@patch('evennia.utils.gametime.gametime', new=Mock(return_value=2975000766))
```
Comment (Griatch) [inline evennia/contrib/tests.py]: Nice. :)

File: evennia/utils/tests/test_gametime.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+from unittest.mock import Mock
+
+from django.conf import settings
+from django.test import TestCase
+
+from evennia.utils import gametime
+
+
+class TestGametime(TestCase):
+    def setUp(self) -> None:
```
Comment (Griatch) [inline evennia/utils/tests/test_gametime.py]: First time this style of annotation is used in the Evennia code. I still have not gotten used to this (I think it looks pretty messy), but I guess it's fine here.

**Pull request merged**